fetch-github-issues/fetch-github-issues.sh

```diff
@@ -5,8 +5,20 @@ set -eEuo pipefail
 PROJECT_ID=1
 OWNER_NAME=gristlabs
 
-fetch_column() {
-  gh project item-list $PROJECT_ID --owner $OWNER_NAME --format json -L "$item_count" -q ".items[] | select(.status == \"$1\")" | jq -c
+output=$(mktemp --suffix=".issues-gh.json")
+trap 'rm -f "$output"' EXIT
```

Comment:
I suggest to remove this line, in case of error, we may would like to track the output.

Also it does not contain sensible data nor it should weight a lot.